web3j-ext/README.md

```diff
@@ -16,8 +16,9 @@ repositories {
 }
 
 dependencies {
-    implementation "foundation.klaytn:web3j-ext:v0.9.0"
-    implementation "foundation.klaytn:web3rpc-java:v0.9.0"
+    implementation "foundation.klaytn:web3j-ext:v0.9.2"
+    implementation "foundation.klaytn:web3rpc-java:v0.9.2"
```

Comment:
Didn't you decide to leave it with v0.9.0?

Reply:
What I intended with v0.9.0 is for the web3j-ext build and publish environment, but for consistency, I will guide users who use web3j-ext and web3rpc-java to use web3rpc-java:0.9.0 as well.